apps/dotcom/src/routes.test.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from '@jest/globals'
+import type { MatcherFunction } from 'expect'
+import { RouteObject } from 'react-router-dom'
+import { SPA_ROUTE_FILTERS } from '../spaRouteFilters'
+import { router } from './routes'
+
+const toMatchAny: MatcherFunction<[regexes: unknown]> = function (actual, regexes) {
+	if (
+		typeof actual !== 'string' ||
+		!Array.isArray(regexes) ||
+		regexes.some((regex) => typeof regex !== 'string')
+	) {
+		throw new Error('toMatchAny takes a string and an array of strings')
+	}
+
+	const pass = regexes.map((regex) => new RegExp(regex)).some((regex) => actual.match(regex))
+	if (pass) {
+		return {
+			message: () =>
+				`expected ${this.utils.printReceived(actual)} not to match any of the regexes ${this.utils.printExpected(regexes)}`,
+			pass: true,
+		}
+	} else {
+		return {
+			message: () =>
+				`expected ${this.utils.printReceived(actual)} to match at least one of the regexes ${this.utils.printExpected(regexes)}`,
+			pass: false,
+		}
+	}
+}
+
+expect.extend({ toMatchAny })
```

Comment:
adding a custom jest matcher is a neat way to make errors more readable for these tests, nice idea.